### PR TITLE
Setup backend and add usage documentation

### LINKED ISSUE #14: Set up MongoDB for project
**Describe the feature to implement.**
Set up MongoDB for the project. This will involve a lot of research. I will research the process for setting up Mongo and add more details to this page. Additionally, we will have to do the schema design for the project. I will create a meeting for that once Mongo is setup.
**Figma Screenshot if applicable**

**Additional context if applicable**

### PR DESCRIPTION
**Pull request by AmanKAggarwal**
### Adminstrative Info
[closes #14] Complete setup for backend

### Changes
What changes did you make?
- Updated backend/README.md with all instructions to get started with backend development (Please check it out
- Finished setting up mongoDB on Atlas.
- Added sample endpoint for review

### Testing
- Tested the mongoDB connection using a sample /api/add-user endpoint

### Confirmation of Change
https://youtu.be/u0cUlLhZiaE

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,42 @@ npm start
...
+1. Running live development server: `npm run dev` (Recommended)
+2. Start the server without live refresh: `npm start`
+
+Location at which server will be hosted will be displayed on terminal. By default, it will be live at http://localhost:3000.
+
+We are using a mongoDB database named "LAK" hosted on MongoDB Atlas. Please ping Assaf to get yourself added to the database so that you can start working on the backend. 
+
+Here is the backend repo structure
+
+```bash
```
Comment (AmanKAggarwal) [inline README.md:48]: Please review this repo layout and let us know if you want it changed.

File: backend/models/user.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+// const bcrypt = require("bcrypt"); Import when needed
```
Comment (AmanKAggarwal) [inline backend/models/user.js:1]: bcrypt hashing can be used while implementing authentication @doraemon127 
Check TSE Octavian for reference [link](https://github.com/TritonSE/TSE-Oktavian/blob/main/backend/models/user.js)

File: backend/routes/index.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+const routes = require('express').Router();
+const User = require('../models/user');
+
+routes.get('/hello', (req, res) => { // Only for testing
+  res.json({message: 'Hello world! Lakta here'})
+})
+
+routes.post('/add-user', (req, res) => { // Only for testing
```
Comment (AmanKAggarwal) [inline backend/routes/index.js:8]: Sample endpoint you can use to test the mongoDB setup

**Comment by AmanKAggarwal** [comment]
@NirmalAgnihotri Should we setup ESLint?

File: README.md (hunk 1 above)
Comment (NirmalAgnihotri) [reply README.md:48]: Yes, this looks great! very well organized.

File: backend/models/job.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
+// noinspection SpellCheckingInspection
+
+const mongoose = require("mongoose");
+
+const { Schema } = mongoose;
+
+const JobSchema = new Schema({
```
Comment (NirmalAgnihotri) [inline backend/models/job.js:7]: Not sure if we already discussed this, but how are we keeping track of the final driver that is chosen for the jobs. Perhaps we can add a field.

**Review by NirmalAgnihotri — APPROVED** [review]
Overall this looks great! Just look into my comment about the final assigned driver.

File: backend/models/job.js (hunk 4 above)
Comment (aworld1) [reply backend/models/job.js:7]: This has been added as a non required field in my latest commit

File: README.md (hunk 1 above)
Comment (doraemon127) [reply README.md:48]: Looks good! Very easy to follow :)

File: backend/models/user.js (hunk 2 above)
Comment (doraemon127) [reply backend/models/user.js:1]: I'll check this out, thanks!

**Review by doraemon127 — APPROVED** [review]

**Pull request merged**